vm/nomad-job.nix

```diff
@@ -120,7 +120,7 @@ ${''
           }
         '') config.microvm.interfaces}
 
-        ${lib.concatMapStrings (share@{ tag, source, socket, proto, ... }:
+        ${lib.concatMapStrings (/*share@*/{ tag, source, socket, proto, ... }:
```

Comment:
If these are unused, feel free to remove. Btw, do you know https://github.com/astro/deadnix?